src/components/styles.css

```diff
@@ -379,6 +385,20 @@
   opacity: 1;
 }
 
+.error {
+  display: none;
+  margin-top: 20px;
+  text-align: center;
```

Comment:
- [ ] Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). kindly change the background-color of your navbar, footer, and main page, ... to match the original design. Make sure the main page is aligned in the center, you need to fix the margin-left and margin-right and you need to add the missing part 👍🏻 

- Your Design:

![18](https://user-images.githubusercontent.com/79058364/191857527-7b5fb695-a663-4b21-b1ea-4d7c617cfc55.png)

- Original Design:

![image](https://user-images.githubusercontent.com/79058364/191857667-6f501420-6d4a-41ab-a615-f0829889a246.png)